### PR TITLE
Update application.conf to increase the cap of WebSocket messages

### PR DESCRIPTION
**Pull request by DeyuChen**
Increase the cap of WebSocket messages

### CODE REVIEW

File: cloudberry/neo/conf/application.conf [hunk 1]
```diff
@@ -93,3 +93,5 @@ asterixdb.url = "http://localhost:19002/query/service"
 asterixdb.lang = SQLPP
 #asterixdb.lang = sparksql
 asterixdb.view.meta.name = "viewMeta"
+
+play.websocket.buffer.limit = 5M
```
Comment (waans11) [inline cloudberry/neo/conf/application.conf]: Maybe put a comment here? Why 5M needs to be allocated?

Comment (chenlica) [inline cloudberry/neo/conf/application.conf]: Agreed, especially since so much effort was put to this one-line change :-)

Comment (DeyuChen) [inline cloudberry/neo/conf/application.conf]: Actually, for TwitterMap example, 5M might be a bit too much. I just put a safe value for future cases.

**Comment by waans11** [comment]
LGTM

**Pull request merged**